.circleci/config.yml

```diff
@@ -283,6 +315,22 @@ jobs:
       - *addsshkeys
       - *commitworkstationdebs
 
+  build-nightly-buster-securedrop-export:
+    docker:
+      - image: circleci/python:3.7-buster
+    steps:
+      - checkout
+      - *installdeps
+      - *fetchwheels
+      - *clonesecuredropexport
+      - *getnightlyversion
+      - *makesourcetarball
+      - *updatedebianchangelog
+      - *builddebianpackage
+      - *installgitlfs
+      - *addsshkeys
+      - *commitworkstationdebs
```

Comment:
the run step here has stretch specific logic so we'll need to inspect the platform we're on via `lsb_release -sc` to determine which directory to put the built deb in as we are doing in the `updatedebianchangelog` run step